src/kubearchive/const.ts

```diff
@@ -0,0 +1 @@
+export const KUBEARCHIVE_PATH_PREFIX = 'plugins/kubearchive';
```

Comment:
nit: with Michal's PR merged, we're hardcoding `'plugins/kubearchive'` in `src/shared/components/pipeline-run-logs/logs/Logs.tsx` and `src/shared/components/pipeline-run-logs/logs/logs-utils.ts` files. It might be worth it to update them to use the `KUBEARCHIVE_PATH_PREFIX` const :)